### PR TITLE
Add Cloud Client NL API samples.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jmdobry** [comment]
Blocked by GoogleCloudPlatform/google-cloud-python#2747

File: language/cloud-client/snippets.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    # Instantiates a plain text document.
+    document = language_client.document_from_text(text)
+
+    # Detects entities in the document. You can also analyze HTML with:
+    #   document.doc_type == language.Document.HTML
+    entities = document.analyze_entities()
+
+    for entity in entities:
+        print('=' * 20)
+        print('         name: %s' % (entity.name,))
```
Comment (jonparrott) [inline language/cloud-client/snippets.py]: don't use `%`,

also, you could do:

```python
print('name: {:>10}'.format(entity.name))
```

https://pyformat.info/#string_pad_align

File: language/cloud-client/snippets_test.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```
Comment (jonparrott) [inline language/cloud-client/snippets_test.py]: This shouldn't be needed (this file contains no extended unicode characters)

Comment (jmdobry) [inline language/cloud-client/snippets_test.py]: Done.

File: language/cloud-client/snippets.py (hunk 1 above)
Comment (jmdobry) [inline language/cloud-client/snippets.py]: Oops, copied this from the google-cloud-python docs. Fixed.

File: language/cloud-client/snippets_test.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+def test_entities_text(cloud_config, capsys):
+    snippets.entities_text('President Obama is speaking at the White House.')
+    out, _ = capsys.readouterr()
+    assert 'name: Obama' in out
+
+
+def test_entities_file(cloud_config, capsys):
+    snippets.entities_file('gs://bucket/file.txt')
+    out, _ = capsys.readouterr()
+    assert 'Polarity: 1' in out
```
Comment (gguuss) [inline language/cloud-client/snippets_test.py]: s/Polarity/Score

File: language/cloud-client/snippets_test.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def test_syntax_text(cloud_config, capsys):
+    snippets.syntax_text('Hello, world!')
+    out, _ = capsys.readouterr()
+    assert 'Polarity: 1' in out
```
Comment (gguuss) [inline language/cloud-client/snippets_test.py]: Should be score?

File: language/cloud-client/snippets_test.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+# limitations under the License.
+
+
+import snippets
+
+
+def test_sentiment_text(cloud_config, capsys):
+    snippets.sentiment_text('Hello, world!')
+    out, _ = capsys.readouterr()
+    assert 'Polarity: 1' in out
```
Comment (gguuss) [inline language/cloud-client/snippets_test.py]: s/Polarity/Score

File: language/cloud-client/snippets.py [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    language_client = language.Client()
+
+    # Instantiates a plain text document.
+    document = language_client.document_from_text(text)
+
+    # Detects sentiment in the document. You can also analyze HTML with:
+    #   document.doc_type == language.Document.HTML
+    sentiment = document.analyze_sentiment()
+
+    print('Polarity: {}'.format(sentiment.polarity))
```
Comment (gguuss) [inline language/cloud-client/snippets.py]: Replace Polarity with Score

Comment (jmdobry) [inline language/cloud-client/snippets.py]: Done.

File: language/cloud-client/snippets_test.py (hunk 4 above)
Comment (jmdobry) [inline language/cloud-client/snippets_test.py]: Done.

File: language/cloud-client/snippets_test.py (hunk 3 above)
Comment (jmdobry) [inline language/cloud-client/snippets_test.py]: Done.

**Comment by jmdobry** [comment]
Ready for final review. Need `test.txt` uploaded to whatever storage bucket is used by this repo's tests.

**Pull request merged**